apax/md/ase_calc.py

```diff
@@ -182,7 +183,7 @@ def initialize(self, atoms):
             model = make_ensemble(model)
 
         for transformation in self.transformations:
-            model = transformation.apply(model, self.n_models)
+            model = transformation.apply(model)
```

Comment:
I guess this parameter was moved inside the model?

Reply:
Same with the code below.

Reply:
it's now detected in the transformation itself.